__mocks__/openmrs-esm-framework.mock.tsx

```diff
@@ -53,3 +57,5 @@ export const mockPatientId = mockPatient.id;
 export const showToast = jest.fn();
 
 export const ExtensionSlot = ({ children }) => <>{children}</>;
+
+export const navigate = jest.fn();
```

Comment:
At some point let's delete this file and replace the line in the Jest config with `@openmrs/esm-framework/mock`. No need to include it in this PR if it doesn't work right off the bat.